src/frontend/js/components/CourseProductCourseRuns/EnrollableCourseRunList.tsx

```diff
@@ -110,14 +110,16 @@ const EnrollableCourseRunList = ({ courseRuns, order }: Props) => {
       if (relatedEnrollment) {
         await enrollment.methods.update({
           is_active: true,
-          course_run: selectedCourseRun.resource_link,
+          course_run: selectedCourseRun.id,
           id: relatedEnrollment.id,
+          was_created_by_order: relatedEnrollment.was_created_by_order,
```

Comment:
Why not forcing it to true ? If the `relatedEnrollment` already exists and is re-activated through an order, then it must be considered to be created from an order, nope ?

Reply:
Indeed !